features/hybrid/promotion/applying_catalog_promotions/not_reapplying_catalog_promotions_on_variants_once_products_taxon_changes.feature

```diff
@@ -16,14 +16,14 @@ Feature: Not reapplying catalog promotions on variants once the product’s taxo
         And there is disabled catalog promotion "Surprise sale" between "2021-07-01" and "2022-05-04" available in "Web-US" channel that reduces price by "90%" and applies on "Dishes" taxon
         And I am logged in as an administrator
 
-    @api @todo @ui
+    @api @ui @mink:chromedriver
```

Comment:
I encountered issues covering these scenarios without the JS driver, likely due to our current method of accessing Taxons with bootstrap inputs.